src/community/ogcapi/ogcapi-maps/src/main/java/org/geoserver/ogcapi/maps/CollectionDocument.java

```diff
@@ -0,0 +1,75 @@
+/* (c) 2018 Open Source Geospatial Foundation - all rights reserved
```

Comment:
Not sure if you care to have 2020 on these

Reply:
I think they're closely derived from existing code.